### DISCUSSION TITLE
Migrating DKAN datasets to CKAN

### DISCUSSION THREAD

**Discussion by CamHurst** [ckan/ckan#6993] [Q&A]
Hi there

We are currently planning on migrating one of our sites over from DKAN to a CKAN instance and were wondering if there is any easy way to transfer all the current datasets we have on our old site over to CKAN? From searching around it seems some people have attempted this previously but there are no concrete ways listed of how to go about it.
Any help or ideas of how to do this would be most appreciated!

**Comment by kikislater** [comment]
Did you manage to make your migration. I would like to know some feedback about it
There is this repo: 
https://github.com/the-vladman/ckanext-dkan
But it seems outdated now
